tests/test_FBbasis3D.py

```diff
@@ -705,3 +705,23 @@ def testFBBasis3DExpand(self):
                 atol=utest_tolerance(self.dtype),
             )
         )
+
```

Comment:
If I am not mistaken I think we can parametrize them now and just skip the cases that are not `L=8`.  

That would exchange the workarounds below for an `if self.L != 8: SkipTest` or something similar.

The 3D tests used to run for both float32 and float64.  Check that's still the case first. If it isn't we can discuss what to do here.

If 3D end up with their own `param` tuple list, then we might be able to just forgo the branch anyway.

Reply:
They do run for float32 and float64 so we're good on that.

I took a stab at doing what you said (just updated this branch)

What I ran into here is that if I use all the same conditions as we have for `FBbasis2D` (i.e. resolutions up to 32 for each dtype), the `FBbasis3D` test gets much longer even with skips. I tested it and found that a lot of that is just from instantiating the large resolution `FBbasis3D` objects that don't get used. 

If we give `FBbasis3D` a smaller list of parameters (just `L=8`) the speed is greatly improved. But then we'd have a `basis_params_3d` that would be smaller for now but eventually expanded. What do you think? I

Reply:
41 seconds with skipping (with all the parameters)
1.5 seconds with skipping (giving just the L=8 parameters), which is the same timing as in develop

Reply:
`basis_params_3d` sounds good.  I think we would want to keep them separate anyway.  That's what I meant by the comment around FFB2D:

>It might be simplest to have a basis_params_2d and basis_params_3d that live in _basis_utils and are just imported into FB2D, FFB2D, FB3D and FFB3D. Perhaps I overlooked something that would block this.

Let's see how that looks. If you run into more problems we can bail on the 3D components for now. Thanks